common-api/src/main/java/org/pmiops/workbench/cdr/dao/SurveyModuleDao.java

```diff
@@ -16,17 +16,17 @@
   @Query(nativeQuery=true,value="select m.name, m.description,\n" +
       "m.concept_id, COUNT(DISTINCT c.concept_id) as question_count,\n" +
       // We don't show participant counts when filtering by keyword, and don't have a way of computing them easily; return 0.
-      "0 participant_count\n" +
+      "0 participant_count, m.order_number \n" +
       "from survey_module m\n" +
       "join achilles_results r on m.concept_id = r.stratum_1\n" +
       "join concept c on r.stratum_2 = c.concept_id\n" +
       "where r.analysis_id = 3110\n" +
       // Because we're using a native query we use MySQL match() here directly instead of matchConcept()
       // TODO: add AchillesResults entity, replace this with JQL
-      "and (match(c.concept_name, c.concept_code, c.vocabulary_id, c.synonyms) against (?1 in boolean mode) > 0 or\n" +
+      "and (match(c.concept_name) against (?1 in boolean mode) > 0 or\n" +
```

Comment:
The angular view filters only on the concept_name / stratum 4. So changed this to avoid difference in the counts and the result count.

Reply:
https://github.com/all-of-us/workbench/blob/master/public-ui/src/app/views/survey-view/survey-view.component.ts#L166